### ISSUE TITLE
Unable to select elements in ios

### ISSUE THREAD

**Issue by chetnaag95** [appium/appium#19353]
### Do I have the most recent component updates?

- [X] I use the most recent available driver/plugin and server versions

### Is the component officially supported by the Appium team?

- [X] I have verified the component repository is present under the Appium organization in GitHub

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Hi I am trying get text attribute value for an element which I am able to select for android in appium but for ios I am not getting text attribute for that element. Also I am unable to select the particular element. Elements are selects together 

Desired Capability - 
{
  "platformName": "iOS",
  "appium:platformVersion": "16.2",
  "appium:deviceName": "iPhone 14",
  "appium:automationName": "XCUITest",
  "appium:udid": "81A04C71-79DE-4137-A8AE-E6A875C24B8B",
}

### Expected Behavior

Should be able to select particular text element inside scroll view and get text attribute in appium for ios

### Minimal Reproducible Example

Android-
Able to select the study element alone. Also getting text attribute
<img width="473" alt="Screenshot 2023-10-30 at 11 31 54 AM" src="https://github.com/appium/appium/assets/82748928/7a50bf68-6941-45a0-94c1-a9c8f514d1ae">

<img width="214" alt="Screenshot 2023-10-30 at 11 32 07 AM" src="https://github.com/appium/appium/assets/82748928/1b654812-6c42-47db-9a57-4206f5d8dd12">

iOs-
Able to select the study element alone. Not getting text attribute 

<img width="463" alt="Screenshot 2023-10-30 at 11 31 36 AM" src="https://github.com/appium/appium/assets/82748928/a0f21300-daf6-4ffc-9fd3-110d26fef599">

### Environment

- Operating system:
- Appium server version (output of `appium --version`): v2.0.1
- Appium driver(s) and their version(s): xcuitest@4.27.0, uiautomator2@2.25.0
- Appium plugin(s) and their version(s):
- Node.js version (output of `node --version`): v16.20.2
- `npm` version (output of `npm --version`) :8.19.4 
- Last component(s) version which did _not_ exhibit the problem:
- Platform and version under test: react native : 0.67.4
- Real device or emulator/simulator: emulator/simulator

**Comment by chetnaag95** [comment]
@mykola-mokhnach

**Comment by mykola-mokhnach** [comment]
I don't think we can help there. All elements and their attributes are defined by the application accessibility. And the latter could only be modified by app developers.

**Comment by chetnaag95** [comment]
I am mainly facing the selection issue of the element as mentioned in the screenshot above

**Comment by chetnaag95** [comment]
@mykola-mokhnach Can you please help on the details mentioned above

**Comment by mykola-mokhnach** [comment]
Like I said I cannot. Only app developers can
